### DISCUSSION TITLE
issue when i try to open specific Chromium profile with seleniumbase

### DISCUSSION THREAD

**Discussion by idabough** [seleniumbase/SeleniumBase#3347] [General]
hi, sorry for the dumb question, but when i try to open specific Chromium profile with seleniumbase doesn't work for me.
this is my simple code:

```
from seleniumbase import SB
import time
import os

def open():
    Profile_Path = fr"C:\Users\{os.getlogin()}\profiles\ConnectBoite\Profile12" # C:\Users\{os.getlogin()}\AppData\Local\Chromium\User Data\Profile 12
    Bnry_Path = fr"C:\Users\{os.getlogin()}\browser\chrome.exe"
    with SB(user_data_dir=Profile_Path,binary_location=Bnry_Path) as sb:
        url = "https://pixelscan.net"
        sb.open(url)
        sb.sleep(5*60)

if __name__ == '__main__':
   open()
```

i used regular path and custom path for my profile location and i remove space in the custom path but doesn't work, i'm found 'default 'added to the path.
 can someone help me please and thank u

![Screenshot 2024-12-17 210434](https://github.com/user-attachments/assets/600dc846-f7e9-4b75-b013-8ae4fe99a1ca)

**Comment by mdmintz** [comment]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2596#issuecomment-2000160739 / https://github.com/seleniumbase/SeleniumBase/issues/2432#issuecomment-1892459152 / https://github.com/seleniumbase/SeleniumBase/discussions/2331#discussioncomment-7730581.

If you need multiple profiles, set a different `user_data_dir` for each one that you want. Only the `Default` profile is supported.

If you're setting the `user_data_dir` while using UC Mode, then make sure you read https://github.com/seleniumbase/SeleniumBase/discussions/2118#discussioncomment-7029201 first. (Don't mix a regular `user_data_dir` with one that is used by UC Mode Chrome, as the two formats are not compatible.)

**Reply by idabough** [reply]
understood, thank you sir for your time & help
